.github/files/setup-wordpress-env.sh

```diff
@@ -69,8 +70,19 @@ for PLUGIN in projects/plugins/*/composer.json; do
 		echo 'Platform reqs pass, running `composer install`'
 		composer install
 	else
-		echo 'Platform reqs failed, running `composer update`'
-		composer update
+		TMP=$(diff <(composer info --locked --no-dev --format=json | jq -r '.locked[].name' | sort) <(composer info --locked --format=json | jq -r '.locked[].name' | sort) | sed -n 's/^> //p' || true)
```

Comment:
```suggestion
		# Comparing output of `composer info --locked` for packages with and without dev dependencies,
		# then figuring out if the lists are different using `diff`.
		# The output of diff gets worked on by `sed` - picking and printing only lines that start with `> `.
		# Adding || true makes sure the exit code stays 0.
		TMP=$(diff <(composer info --locked --no-dev --format=json | jq -r '.locked[].name' | sort) <(composer info --locked --format=json | jq -r '.locked[].name' | sort) | sed -n 's/^> //p' || true)
```

Reply:
I reworded the suggested comment a bit, how does 190c6c6ca0 look to you?